test/test-usage.js

```diff
@@ -1,68 +1,52 @@
-var assert = require('assert');
-var async = require('async');
+'use strict';
+
+var concat = require('concat-stream');
 var debug = require('debug')('strong-pm:test');
+var executor = require.resolve('../bin/sl-pm.js');
+var fork = require('child_process').fork;
```

Comment:
Seems to me you could do this with `execFile` without pulling in `concat-stream`, unless there's something I'm missing by viewing as a patch?